src/renderer/components/FtKeyboardShortcutPrompt/FtKeyboardShortcutPrompt.vue

```diff
@@ -115,6 +115,14 @@ const localizedShortcutNameDictionary = computed(() => {
     ['SHOW_SHORTCUTS', t('KeyboardShortcutPrompt.Show Keyboard Shortcuts')],
     ['HISTORY_BACKWARD', t('KeyboardShortcutPrompt.History Backward')],
     ['HISTORY_FORWARD', t('KeyboardShortcutPrompt.History Forward')],
+    ...(
+      isMac
+        ? [
+            ['HISTORY_BACKWARD_ALT_MAC', t('KeyboardShortcutPrompt.History Backward (Mac only)')],
+            ['HISTORY_FORWARD_ALT_MAC', t('KeyboardShortcutPrompt.History Forward (Mac only)')],
+          ]
+        : []
+    ),
```

Comment:
**suggestion (non-blocking):** I'm curious if showing a non-monospace slash character joining the different alts would be more elegant than separate entries for alts. I was thinking about storing alts by storing the shortcut constant as `A|B`, which is then parsed into `A<span>/</span>B`, where we can then apply a separate styling rule (no monospace) to the nested `span`s. But that would need some special logic in the caller as well for Mac-only additional shortcut cases like this one, or a separate character for representing non-Mac and Mac-exclusive shortcut alts (e.g., `A┛B┗C|D for Windows/Mac-specific alt B, Mac-specific alt C, and universal alt D).

Reply:
Way too complicated to understand
Left for another PR at least -_-

Reply:
Sounds good - this will currently be the only action with two displayed shortcuts we have in the modal (I excluded the dupe alt shortcuts in the original PR), so I suggest we move to a more elegant solution when we want to add more of them